### PR TITLE
DOCS Authentication

### PR DESCRIPTION
**Pull request by Firesphere**
Replaces #7008

### CODE REVIEW

File: docs/en/02_Developer_Guides/09_Security/00_Member.md [hunk 1]
```diff
@@ -93,39 +90,40 @@ Using inheritance to add extra behaviour or data fields to a member is limiting,
 class. A better way is to use role extensions to add this behaviour. Add the following to your
 `[config.yml](/developer_guides/configuration/configuration/#configuration-yaml-syntax-and-rules)`.
 
-	:::yml
-	Member:
-	  extensions:
-	    - MyMemberExtension
+```yml
+Member:
```
Comment (robbieaverill) [inline docs/en/02_Developer_Guides/09_Security/00_Member.md]: Existing issue, but this should be `SilverStripe\Security\Member`

Comment (Firesphere) [inline docs/en/02_Developer_Guides/09_Security/00_Member.md]: Good catch, should've been updated by now :) (multiple places)

**Comment by Firesphere** [comment]
Please rebuild. I can't imagine some `.md` files breaking the build 😂

File: docs/en/04_Changelogs/4.0.0.md [hunk 2]
```diff
@@ -1391,7 +1392,7 @@ specific functions.
 
 #### Upgrading custom Authenticators
 
-The methods `register` and `unregister` on `Authenticator` are deprecated in favor of the `Config` system. This means that any custom Authenticator needs to be registered through the yml config:
+The methods `register` and `unregister` on `Authenticator` are deprecated in favor of the `Config` system. This means that any custom Authenticator needs to be registered through the yml config. For further information on how to create a custom authentication method, [see the Authention documentation](/developer_guides/Security/Authentication).
```
Comment (robbieaverill) [inline docs/en/04_Changelogs/4.0.0.md]: typo in "authentication"

Comment (Firesphere) [reply docs/en/04_Changelogs/4.0.0.md]: fixed

**Pull request merged**